src/pages/Login.vue

```diff
@@ -28,17 +28,21 @@
 
 <script>
     import Store from '../store.js';
-
-
     export default {
         name: 'login',
         data: function () {
             return {
                 username: null,
                 password: null,
-                remember: true
+                remember: true,
+                errorMessage : Store.errors[0]
```

Comment:
oddly, i dont think that [0] is necessary here. unless you *really are* watching the first error message in the array?